### ISSUE TITLE
Download order needs to receive an argument to set destination path

### ISSUE THREAD

**Issue by palmoreck** [CONABIO/antares3#28]
```antares download_order command``` downloads apparently all orders with column value "downloaded=false" in table madmex_order of database without checking credentials in ```.antares``` file. 

Also the files are downloaded to ~ directory of user. 

Adapt this command to address this cases

**Comment by loicdtx** [comment]
1 - Even better if there's an option to send the downloaded files directly to a bucket (maybe by downloading as in-memory file object, and writing to bucket using `boto3.Client.upload_fileobj()`)

2 - Right now files are downloaded to the global variable `TEMP_DIR`, which  needs to be set in `.antares`. This is not the preferred way to handle temp dirs in UNIX. See [gettempdir](https://docs.python.org/3.5/library/tempfile.html#tempfile.gettempdir). [Most uses](https://github.com/CONABIO/antares3/search?p=1&q=TEMP_DIR&unscoped_q=TEMP_DIR) of TEMP_DIR can probably be replaced by `gettempdir()`, and the `.antares` entry removed.

**Comment by palmoreck** [comment]
2.- Ok, but there's no TEMP_DIR set in ```.antares``` in container of nodo5:

```cat .antares```

```
SECRET_KEY
DEBUG
DJANGO_LOG_LEVEL
DATABASE_NAME
DATABASE_USER
DATABASE_PASSWORD
DATABASE_HOST
DATABASE_PORT
ALLOWED_HOSTS
SERIALIZED_OBJECTS_DIR
INGESTION_PATH
BIS_LICENSE
USGS_USER
USGS_PASSWORD
```

**Comment by palmoreck** [comment]
2. Q: if TEMP_DIR it's not set the files will be downloaded to ~? because there's no TEMP_DIR variable in ```.antares``` of nodo5's container

**Comment by loicdtx** [comment]
That's right, default value (when not set in .antares) is defined in settings.py
